go-install/windows.md

```diff
@@ -1,7 +1,42 @@
 # Setup instructions for Windows
 
-**We recommend using Windows Subsystem for Linux (WSL).**
+**We recommend using Windows Subsystem for Linux 2 (WSL2). No official support for native Windows will be provided.**
 
-1. If you already have WSL setup you can skip this step. Setup WSL by following steps 1, 2 and 3 from [Tilo's Windows setup guide](http://people.cs.bris.ac.uk/~burghard/setup/windows.html).
+## WSL2
 
-2. Open a new Ubuntu terminal and follow the [linux setup guide](./linux.md) to install Go.
+1. Follow the [official instructions here](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to install WSL2. **Do not install WSL1**: some tools will not work correctly. When prompted to install a distribution of your choice, we officially recommend [Ubuntu 20.04 LTS](https://www.microsoft.com/store/apps/9n6svws3rx71).
+
+1. If you already have an existing WSL1 distribution, you can update it to use WSL2 with:
+    ```ps
+    wsl --list --verbose 
+    wsl --set-version DISTRO 2
+    ```
+    where `DISTRO` is the name of the distribution to update.
```

Comment:
might be worth saying how you find out the `DISTRO`

Reply:
That's what `wsl --list --verbose` does but perhaps should make it more clear?